autoware_adapi_v1_msgs/geometry/msg/Shape.msg

```diff
@@ -0,0 +1,18 @@
+# constants
+uint8 UNKNOWN = 0
+uint8 POINT3D = 1
+uint8 POINT2D = 2
+uint8 LINE3D = 3
+uint8 LINE2D = 4
+uint8 POLYGON3D = 5
+uint8 POLYGON2D = 6
```

Comment:
I want list and strip.

Reply:
It can be used as both by `sizes` field.
https://github.com/isamu-takagi/autoware_adapi_msgs/blob/feat/geometry-shape/autoware_adapi_v1_msgs/geometry/shape.md#line3d-and-line2d